### PR TITLE
Add Why estimation is necessary blog

### PR DESCRIPTION
**Pull request by 2KAbhishek**
## What

- [x] Added Why estimation is necessary for software development and how to do it

# Checklist

- [x] My code follows the style guidelines of this project and is properly formatted
- [x] I have performed a self-review of my own changes
- [x] All existing tests pass, and I have added new ones if required

### CODE REVIEW

File: content/blog/esdt.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+slug = "estimation"
+tags = ["Agile", "Estimation"]
+title = "Estimating Software Development Time"
++++
+
+What is the need for an “Estimate”, because everything is bound to change rite in actuality. Before coming to the need for estimation let us set the context for this discussion. Estimation is nothing, but a human being’s innate necessity of wanting to predict the future with the term “Accuracy” directly relating to the probability of the expected outcome. In simple terms, we want stuff to happen the way we want and when we want.
+
+But the reality is seldom this simple, because of the randomness of the events that surround us. It is very difficult to predict the probability of every possible outcome with a certain assumed degree of accuracy.
+
+This is where our buddy “Estimate” comes into the picture. In any line of work, an important question that requires an answer on priority is how soon a project or work or task can be completed because we wanted it to be completed by yesterday!
```
Comment (sapanparikh18) [inline content/blog/esdt.md:17]: How soon and in what budget both

File: content/blog/esdt.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+Let’s get real. Estimations are required because we have to start somewhere and most importantly, to avoid “Waste”. Waste comes in different forms. It can be Natural Resources, Human Resources, Time, Cost, etc.
+
+```
+“The difference between what I think is valuable and what really is valuable creates waste.”
```
Comment (sapanparikh18) [inline content/blog/esdt.md:22]: Pure love here!

File: content/blog/esdt.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+A parameter with its assumed correlation to the amount of time taken is determined and is then scaled up to the actual project size. A deterministic result is a certain number, or a figure based on such scaling of the correlation. A probabilistic result on the other hand provides a range of estimates over a probability density curve as given below:
+
+<image src="/static/images/2022/05/Probabilistic_Parametric_Estimate.png"/>
+
+The first step in this technique is to determine the required parameter and the level of accuracy of the estimate. For example, take the method of story point estimation. We know that 1 story point is assigned to a task or a feature, which is the most basic work that can be completed in a given time. Keeping such a story as a reference, all the other stories can be assigned points. This means we can correlate the time taken to complete a task and the story point, which is
```
Comment (sapanparikh18) [inline content/blog/esdt.md:53]: single digit number should be written in words. So 1 = One

**Review by sapanparikh18 — COMMENTED** [review]
An Amazing blog Arjun!
Hand just one or two nitpicky comments which I have added. 
Though apart from that can you add two more aspects to the whole estimate aspect of an agile project?
1. Trust: The less you trust the team the more you feel the need to get estimates (I would like to know your thoughts on this one) Also, how the long cycle times in releases tend to change the estimates and thus depletes trust
2. No Estimate: Recently, how high-functioning teams have started working totally without estimates because they keep the batch size extremely small. They slice the stories into a very thin quantum of work and they add value on daily bases, if not more frequently.

**Review by hawkeyes21 — APPROVED** [review]

**Pull request merged**